cpp/include/cudf/aggregation.hpp

```diff
@@ -110,8 +119,9 @@ class aggregation {
     COLLECT_SET,     ///< collect values into a list without duplicate entries
     LEAD,            ///< window function, accesses row at specified offset following current row
     LAG,             ///< window function, accesses row at specified offset preceding current row
-    PTX,             ///< PTX  UDF based reduction
-    CUDA,            ///< CUDA UDF based reduction
+    PTX,             ///< PTX  based UDF aggregation
+    CUDA,            ///< CUDA based UDF aggregation
+    HOST_UDF,        ///< host based UDF aggregation
```

Comment:
Nit: Put the `HOST_UDF` at the last of the list, we will update less code.

Reply:
As mentioned before, we should group the UDF aggregations together to reduce fragmentation. So this one should stay close to `PTX` and `CUDA` aggregations.